### PR TITLE
Surya/core 10832/nopw cmdsignup

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

**Comment by heronhaye** [comment]
engine/logout.go
engine/account_delete.go
engine/paperprovision.go
engine/deprovision.go
engine/login.go
engine/login_provision.go
engine/signup.go

**Comment by zapu** [comment]
are these other engines we need to change?

File: go/client/util.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+func ensureSetPassphraseFromRemote(mctx libkb.MetaContext) error {
+	if !mctx.G().ActiveDevice.Valid() {
+		return nil
+	}
+	cli, err := GetUserClient(mctx.G())
+	if err != nil {
+		return err
+	}
+	ret, err := cli.CanLogout(mctx.Ctx(), 0)
```
Comment (zapu) [inline go/client/util.go]: btw this already does the `!h.G().ActiveDevice.Valid()` call

Comment (heronhaye) [reply go/client/util.go]: good catch, i'll remove it here then

**Comment by heronhaye** [comment]
merging over autoreset flake

**Pull request merged**